README.md

```diff
@@ -3,19 +3,25 @@ ARMO Vulnerability Scanning
 
 
 # Installing ARMO cluster components in a Kubernetes cluster Using Helm:
-=============================================================
 
 1. Add the Vulnerability Scanning Helm Repo
 ```
 helm repo add armo https://armosec.github.io/armo-helm/
 ```
 
-2. Get Your Account ID from ARMO
+2. Update helm repo
 ```
-kubescape config local get customerGUID
+helm repo update
 ```
 
 3. Install the Helm Chart, use your account ID and give your cluster a name 
 ```
-helm upgrade --install armo  armo/armo-cluster-components -n armo-system --create-namespace --set accountGuid=<my_account_guid> --set clusterName=`kubectl config current-context`
+helm upgrade --install armo  armo/armo-cluster-components -n armo-system --create-namespace --set accountGuid=<account_guid> --set clusterName=`kubectl config current-context`
 ```
+
+ 
+## Values
+* `accountGuid` - account ID [required]
+* `clusterName` - cluster name [required]
+* `clientID` - api token client ID 
```

Comment:
can we add "client id" to the helm command?

Reply:
@rotemamsa Can you please explain your question?

After this commit, you can add the clientID like you add the accountGUID. e.g
```
helm upgrade --install armo  armo/armo-cluster-components -n armo-system --create-namespace --set accountGuid=<my_account_guid> --set clusterName=`kubectl config current-context` --set clientID=<generated client id> --set secretKey=<generated secret key>
```